src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -221,8 +221,9 @@ private CommandResult executeCommand(String commandText) throws CommandException
             }
 
             //So is this. Todo: Compile both the above and below into a new "UpdateUI" class.
-
-            if (commandText.equals("list")) {
+            if (commandText.equals("bank")) {
```

Comment:
Perhaps we can move these checks away from MainWindow, so we don't break the abstraction of UI.

Reply:
Ok I believe @dragontho  is figuring a way to restructure the views.